### ISSUE TITLE
UTF8 font?

### ISSUE THREAD

**Issue by rodrigob** [m5stack/uiflow-micropython#25]
I do not see any UTF8 font included. (while Flow 1 had Unicode24 included)

**Comment by lbuque** [comment]
No, because it needs to be compatible with some boards with 4MB flash.

**Comment by rodrigob** [comment]
As a proof of concept I injected the font from examples/Advanced/Display/Display_Unicode into the uiflow2 binary. That seems to work fine, I can now show UTF-8 text on screen. 

 I am stuck with issues regarding the connectivity with UIFlow, so for now I can only load the program via USB.

**Comment by lbuque** [comment]
Your host's flash should be 16MB.

**Comment by lbuque** [comment]
@rodrigob Now you can use vlw font.

![Image](https://github.com/user-attachments/assets/2d15c2d2-eed0-4175-9067-fa99403a9c89)
